internal/impl/kafka/enterprise/integration_test.go

```diff
@@ -200,11 +191,19 @@ max_message_bytes: 1MB
 	license.InjectTestService(conf.Resources())
 
 	gmgr := enterprise.NewGlobalRedpandaManager("foo")
-	require.NoError(t, gmgr.InitWithCustomDetails("meowcustom", logsTopicOverride, statusTopicOverride, &kafka.FranzConnectionDetails{
-		SeedBrokers: []string{brokerAddr},
-		MetaMaxAge:  time.Minute * 5,
-		ClientID:    "foobar",
-	}))
```

Comment:
This was an interesting footgun... We added some more fields which cannot be set to their default Go value.